fetch.bs

```diff
@@ -3684,11 +3788,37 @@ steps:
 
  <li><p>Wait for either <var>response</var>'s <a for=response>body</a> to be null, or
  <var>response</var>'s <a for=response>body</a>'s <a for=body>stream</a> to be
- <a for=ReadableStream>closed</a> or <a for=ReadableStream>errored</a>, and then set
- <var>request</var>'s <a>done flag</a>.
- <!-- This is really bad and needs to be handled differently at some point. -->
+ <a for=ReadableStream>closed</a> or <a for=ReadableStream>errored</a>, and perform the following steps:
+ <ol>
+  <li><p>Set <var>request</var>'s <a>done flag</a>.
+   <!-- This is really bad and needs to be handled differently at some point. -->
+  <li>Call <a for=/>report resource timing</a> with <var>fetchParams</var> and <var>response</var>.
+ </ol>
 </ol>
 
+<h3 id=fetch-report-resource-timing>Report Resource Timing</h3>
```

Comment:
I don't think this needs an `h3` here. This feels like it could be an algorithm that's part of the previous section.

Reply:
ack